Scripts/DCS-BIOS/doc/json/AircraftAliases.json

```diff
@@ -119,6 +120,7 @@
                   "T-45": [ "CommonData", "VNAO_T-45" ],
                 "TF-51D": [ "CommonData", "NS430", "P-51D" ],
                  "UH-1H": [ "CommonData", "NS430", "UH-1H" ],
+                "UH-60L": [ "MH-60R" ],
```

Comment:
Shouldn't this also show Commondata? You likely need to add this to AircraftList.lua as well